cognite/client/data_classes/simulators/routine_revisions.py

```diff
@@ -542,11 +560,11 @@ def _load(cls, resource: dict, cognite_client: CogniteClient | None = None) -> S
             if resource.get("configuration")
             else None
         )
-        script = (
-            [SimulatorRoutineStage._load(stage_, cognite_client) for stage_ in resource.get("script", [])]
-            if resource.get("script")
-            else None
-        )
+        script = None
+        if resource.get("script"):
+            stages = [SimulatorRoutineStage._load(stage_, cognite_client) for stage_ in resource.get("script", [])]
```

Comment:
I hold a personal grudge against trailing underscores 😂 

```suggestion
            stages = [SimulatorRoutineStage._load(stg, cognite_client) for stg in resource.get("script", [])]
```